apps/docs/src/app/platform/component-docs/platform-wizard-generator/examples/wizard-generator-default-example.component.ts

```diff
@@ -13,6 +13,8 @@ export class WizardGeneratorDefaultExampleComponent {
     stepItems: WizardGeneratorItem[] = [
         {
             name: 'Product type',
+            title: 'Product Type Selection',
+            optionalText: 'first step',
```

Comment:
All good, but for consistency, maybe add optional text for all steps in the example?

Reply:
I will update the example. Thanks!